api/app/Http/Controllers/ChildEntityController.php

```diff
@@ -157,7 +158,7 @@ public function putMany(Request $request, $id)
         $childModels = $this->getChildModel()
             ->hydrateRequestCollection($requestCollection, $existingChildModels);
 
-        $this->getRelation($parent)->saveMany($childModels);
+        $this->saveMany($parent, $childModels);
```

Comment:
We shouldn't be adding these methods into the controller, as the revisionable trait is more of a model concern. One option would be to extend the actual relations so the saveMany method emits the desired event, but this would be a lot of work.

We need to rethink how these child entities are being tracked.